File: app/src/main/java/ir/ahfz/rentcar/io/network/model/ExtraResponse.java
package ir.ahfz.rentcar.io.network.model;

import java.util.List;

import ir.ahfz.rentcar.io.network.BaseResponse;

public class ExtraResponse extends BaseResponse {

    private List<Extra> extras = null;

    public List<Extra> getExtras() {
        return extras;
    }

    public void setExtras(List<Extra> extras) {
        this.extras = extras;
    }


    public static class Extra {

        private int id;
        private String title;
        private String description;
        private String price;
        private String createdAt;
        private String updatedAt;

        public int getId() {
            return id;
        }

        public void setId(int id) {
            this.id = id;
        }

        public String getTitle() {
            return title;
        }

        public void setTitle(String title) {
            this.title = title;
        }

        public String getDescription() {
            return description;
        }

        public void setDescription(String description) {
            this.description = description;
        }

        public String getPrice() {
            return price;
        }

        public void setPrice(String price) {
            this.price = price;
        }

        public String getCreatedAt() {
            return createdAt;
        }

        public void setCreatedAt(String createdAt) {
            this.createdAt = createdAt;
        }

        public String getUpdatedAt() {
            return updatedAt;
        }

        public void setUpdatedAt(String updatedAt) {
            this.updatedAt = updatedAt;
        }

    }

}

File: app/src/main/java/ir/ahfz/rentcar/io/model/ExtraResponse.kt
package ir.ahfz.rentcar.io.model

import ir.ahfz.rentcar.io.network.BaseResponse

/**
 * Data model for Extra response
 */
data class ExtraResponse(var extras: List<Extra>? = null) : BaseResponse() {

    /**
     * Data model for Extra
     */
    data class Extra(
        var id: Int = 0,
        var title: String? = null,
        var description: String? = null,
        var price: String? = null,
        var createdAt: String? = null,
        var updatedAt: String? = null
    ) {


    }

}
